internal/engine/build_and_deployer_test.go

```diff
@@ -553,6 +553,27 @@ func TestContainerBuildMultiStage(t *testing.T) {
 	assert.Equal(t, k8s.MagicTestContainerID, result.OneAndOnlyContainerID().String())
 }
 
+func TestDockerComposeFastBuild(t *testing.T) {
+	f := newBDFixture(t, k8s.EnvNone)
```

Comment:
imo this test is extra compelling if you set a k8s Env that would normally do a synclet build, and then asserted that there was no synclet call

Reply:
done!